src/components/BuildBaronAndAnnotations/AddIssueModal.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useReducer } from "react";
+import React, { Dispatch, SetStateAction, useReducer } from "react";
```

Comment:
We dont usually import these types directly 
We normal use them as such `React.Dispatch` `React.SetStateAction` https://github.com/evergreen-ci/spruce/blob/ea864962358c85134c190b7ee9951e1b18cef51f/src/pages/spawn/spawnHost/spawnHostModal/PublicKeyForm.tsx#L21